content/en/metrics/guide/dynamic_quotas.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Dynamic Quotas for Metrics
+further_reading:
+- link: "/account_management/billing/custom_metrics/?tab=countrate"
+  tag: "Documentation"
+  text: "Custom Metrics Billing"
+- link: "/metrics/guide/custom_metrics_governance/"
+  tag: "Guide"
+  text: "Best Practice for Custom Metric Governance"
+- link: "https://www.datadoghq.com/blog/metrics-without-limits/"
+  tag: "Blog"
+  text: "Dynamically control your custom metrics volume with Metrics without Limits™"
+algolia:
+  tags: ['custom metrics']
+is_beta: true
+private: true
+---
+
+## Overview
+
+{{< callout url="#" btn_hidden="true">}}
+  This feature is currently in Preview.
+{{< /callout >}} 
+
+As your business scales, managing observability costs could become unruly quickly. Effective custom metrics governance should increase monitoring efficiency rather than cut visibility entirely. And it should not be a time-consuming, cumbersome process to protect your organization from unintentional billing overages. 
```

Comment:
```suggestion
As your business scales, managing observability costs could become unruly quickly. Effective custom metrics governance should increase monitoring efficiency rather than cut visibility entirely. And it should not be a time-consuming, cumbersome process to protect your organization from unintentional billing overages. As mentioned in Datadog's [Best Practices Guide for Custom Metrics Governance](https://docs.datadoghq.com/metrics/guide/custom_metrics_governance/#monitoring-and-prevention), you can easily monitor spikes in your custom metrics volumes. 

With **Datadog's Dynamic Metrics Quotas**, you now additionally have automatic spike remediation -- reducing unintentional metrics usage and custom metrics costs.
```